src/index.js

```diff
@@ -107,6 +107,9 @@ const init = async () => {
       interactions: config.hostingEnvironment.interactionsUrl,
       assets: assetsUrl,
     },
+    entra: {
+      enableEntraSignIn: config.entra.enableEntraSignIn,
```

Comment:
Can we change this to 
`useEntraForAccountRegistration: config.entra.useEntraForAccountRegistration`